tests/CMakeLists.txt

```diff
@@ -16,3 +16,27 @@ add_executable(tests ${TEST_SOURCES})
 target_include_directories(tests PRIVATE tests)
 target_link_libraries(tests Catch2::Catch2WithMain)
 catch_discover_tests(tests)
+
+set(ctest_test_args --output-on-failure)
+
+include(ProcessorCount)
```

Comment:
this is kind of unnecessary if using e.g. ninja over standard make

Reply:
You added it https://github.com/prusa3d/Prusa-Firmware/commit/5ddac5cb6c725972bdf2f6861f4999ef13fe1193#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0d

Reply:
I'd keep it, it does no harm and allows the good-old-make taking advantage of multiple cores